pages/facts.html

```diff
@@ -1,12 +1,178 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/facts.css"> 
+    <title>Bubble Tea Facts</title>
 </head>
 <body>
+    <!-- NAVBAR STARTS -->
+    <head>
+        <!-- Top navigation -->
+        <nav class="topnav">
+
+      <!-- Centered link -->
+
+      <nav class="topnav-centered">
+        <button href="/pages/index.html">
+          <img id="home" class="active" src="/images/EJY.jpg" />
```

Comment:
In pure HTML, `/>` isn't allowed. It's fine in JSX, though!

```suggestion
          <img id="home" class="active" src="/images/EJY.jpg">
```

Reply:
Also, the W3C would want you to add an `alt` text attribute to this.